client/src/components/Draftail/blocks/EmbedBlock.js

```diff
@@ -10,26 +10,40 @@ import MediaBlock from '../blocks/MediaBlock';
  */
 const EmbedBlock = props => {
   const { entity, onRemoveEntity } = props.blockProps;
-  const { url, title, thumbnail } = entity.getData();
+  const { url, title, thumbnail, providerName, authorName } = entity.getData();
+
+  if (thumbnail) {
```

Comment:
Could you turn this and the other case around? I think I’d rather have `!thumbnail` as a special case, then keep the `return` at the end of the function be the "has thumbnail" scenario.